test_data/all_data/beol-data.ttl

```diff
@@ -168,6 +196,91 @@
   knora-base:hasPermissions "CR knora-admin:Creator|M knora-admin:ProjectMember|V knora-admin:KnownUser|RV knora-admin:UnknownUser";
   knora-base:attachedToUser <http://rdfh.ch/users/PSGbemdjZi4kQ6GHJVkLGF> .
 
+<http://rdfh.ch/0801/XNn6wanrTHWShGTjoULm5g> a beol:letter;
+  rdfs:label "letter to self";
+  knora-base:isDeleted false;
+  knora-base:attachedToProject <http://rdfh.ch/projects/yTerZGyxjZVqFMNNKXCDPF>;
+  knora-base:creationDate "2020-09-18T10:46:06.442289Z"^^xsd:dateTime;
+  knora-base:hasPermissions "CR knora-admin:Creator|V knora-admin:UnknownUser";
+  knora-base:attachedToUser <http://rdfh.ch/users/PSGbemdjZi4kQ6GHJVkLGF>;
+  beol:title <http://rdfh.ch/0801/XNn6wanrTHWShGTjoULm5g/values/jGtT_87FS2qTkUEigdXMPg>;
+  beol:hasSubject <http://rdfh.ch/0801/XNn6wanrTHWShGTjoULm5g/values/YdnovfaXRnW4c5qO7h02Ag>;
```

Comment:
This is test data. Why would it be used on the BEOL live server?

Reply:
beol-data.ttl is not a pure test data. It contains subject index definition as list nodes, etc. The imported BEOL data and the data in this file are combined and made available on production server. for example, you can see that [test letter](https://beol.dasch.swiss/letter/http:%2F%2Frdfh.ch%2F0801%2Fletter_resource) defined in this file has ended up on Beol production server.

Reply:
I understand, but that doesn't seem right to me, because there are already fake letters in `beol-data.ttl`. Wouldn't it be better to make two files, to separate the real data from the test data?

Reply:
I removed the `hasSubject` in 99d13d11.

Reply:
> I understand, but that doesn't seem right to me, because there are already fake letters in `beol-data.ttl`. Wouldn't it be better to make two files, to separate the real data from the test data?

It would have been, but never happened!